app/controllers/ncr/work_orders_controller.rb

```diff
@@ -29,7 +29,7 @@ def update
 
       super
 
-      if @model_changing && !@model_instance.emergency # skip approvals if emergency
+      unless skip_state_update?
```

Comment:
Ha. Well, that does describe (literally) what the method is doing, but I was going for a name that would convey why those things matter. Does that make sense?